providers/gce/gce.go

```diff
@@ -770,8 +770,8 @@ func (g *Cloud) SetInformers(informerFactory informers.SharedInformerFactory) {
 		UpdateFunc: func(prev, obj interface{}) {
 			prevNode := prev.(*v1.Node)
 			newNode := obj.(*v1.Node)
-			if newNode.Labels[v1.LabelFailureDomainBetaZone] ==
```

Comment:
is this safe or should we check for both? i.e. first for `LabelTopologyZone` and if not present fall bak to `LabelFailureDomainBetaZone`

Reply:
we need to keep checking both labels per the documentation

https://github.com/kubernetes/api/blob/faec07c7cc8998ed89b8633ba2bc41b9323a7dc6/core/v1/well_known_labels.go#L26-L34